### ISSUE TITLE
关于解码的问题

### ISSUE THREAD

**Issue by han-xie** [Pelhans/ZASR_tensorflow#4]
Hanpei你好，
请问这个版本的代码目前都是可以跑通的吗？
我在运行这个代码的时候发现训练的时候输出的Transcript都是空的，是因为tf的ctc_decoder使用得有问题吗？
另外，运行python test.py的时候
使用lm_decode解码出来的结果也是乱码~
![zasr](https://user-images.githubusercontent.com/27179209/50133020-ca286000-02c4-11e9-89c8-d7f639da0d01.png)

**Comment by han-xie** [comment]
作者已经邮件回答了我的问题~感谢作者Pelhans。
此模型目前还不收敛，因此也没有一个可用的模型。

另外，关于解码的部分，经过我本人的测试：如果原来使用的是tensorflow的ctc decoder，要换成百度的swig_decoders的话，确实需要加一个softmax函数。实测tensorflow 1.11.0里边要用tf.nn.softmax(self.logits, axis=2)才能得到正确的结果~【这个代码里边在model_utils/init_model.py里边可以找到tf.nn.softmax，用新版的tensorflow的话应该要修改一下~】
